subsys/bluetooth/mesh/transport.h

```diff
@@ -101,6 +101,8 @@ void bt_mesh_trans_init(void);
 
 void bt_mesh_rpl_clear(void);
 
+bool is_replay(struct bt_mesh_net_rx *rx, struct bt_mesh_rpl **match);
```

Comment:
Do you notice how this is the first function name in this header file that doesn't have the bt_mesh prefix? I'd suggest fixing this now so you don't have to do the cleanup later. Since you already have a `bt_mesh_rpl_clear()` I think it'd make sense to follow that prefix, i.e. rename this e.g. to `bt_mesh_rpl_check()` or something like that.